### PR TITLE
Add disable-content-trust noop flag to bud

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Adds a NOOP disable-content-trust flag to buildah bud

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
FYI @baude @mheon

File: pkg/cli/common.go [hunk 1]
```diff
@@ -95,6 +95,10 @@ var (
 			Value: "",
 			Usage: "use `[username[:password]]` for accessing the registry",
 		},
+		cli.BoolFlag{
+			Name:  "disable-content-trust",
+			Usage: "This is a Docker specific option and is a NOOP for Buildah",
```
Comment (rhatdan) [inline pkg/cli/common.go]: Don't say Buildah, since this will also be used in podman.
Just remove for Buildah.

Comment (TomSweeneyRedHat) [reply pkg/cli/common.go]: @rhatdan ack, good point.

**Comment by TomSweeneyRedHat** [comment]
@rhatdatn anything I can/need to do from my side or is homu just not happy today?

**Comment by rhatdan** [comment]
@TomSweeneyRedHat I have no idea, will wait for Travis and then manually merge.

**Comment by rhatdan** [comment]
HOMU is not in a good mood, forcing push.

**Pull request merged**